lib/rules/selector-not-notation/__tests__/index.mjs

```diff
@@ -140,6 +152,81 @@ testRule({
 			endLine: 1,
 			endColumn: 12,
 		},
+		{
+			code: stripIndent`
+				/* a comment */
+				:not(a, ), /* a comment */
+				:not(a, ),
+				/* a comment */
+				:not(a, )
+				{}
```

Comment:
I think the `:not(a, )` test cases are extremely exotic.
Can we rewrite the new tests to use more common patterns?

Maybe something like `:not(a, div)`?

Reply:
Thank you for the clarifications. I've removed in [this commit](https://github.com/stylelint/stylelint/pull/8074/commits/518bd98461e90f990ed85e40afe001efd4e3ff33).

Reply:
Thank you for these further edits 🙇